src/h5web/visualizations/shared/ScreenshotButton.module.css

```diff
@@ -5,7 +5,7 @@
   color: inherit;
 }
 
-.box {
+.btnLike {
```

Comment:
I took inspiration from your `btnAspect` class but I also remembered having seen the naming `fooLike` being used before for this very purpose - i.e. for elements that "look like" other elements. It's quite common for headings, notably (e.g. `h2Like`)